doc: 📝 Update doc to use OVHcloud image (#5940)
ovh/docs: pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md

> 
> A common configuration is to use non-autoscaled, monthly-billed node pools as base for your static workload, and autoscaled, hourly-billed node pools with smaller flavors for your dynamic workload. 

> [!primary]
>
> The source code of the following example is available in the public Github repository [public-cloud-examples](https://github.com/ovh/public-cloud-examples/tree/main/containers-orchestration/managed-kubernetes/autoscaler)

When you create your cluster, you can bootstrap a default node pool in it, and you can add others in the Public Cloud section of the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) or directly [using the Kubernetes API](/pages/public_cloud/containers_orchestration/managed_kubernetes/managing-nodes).

## Deploying a test workload

Let's assume that you have created an MKS cluster with a node pool with its minimum set to 1 and its maximum set to 3.

> In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod simply heavy charge each allocated CPU of your pod. It's a CPU intensive operation, but it uses a minimal amount of memory.

Reviewer: "The *Python CPU load* pod simply heavy charge each allocated CPU of your pod."
Isn't a verb missing here ? I don't understand that sentence.

Author: yes agree :), is it better: "The *Python CPU Load* pod aims to consume all the CPU allocated to it" ?

Reviewer: ```suggestion
In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod's goal is to consume all the CPU allocated to it. It's a CPU intensive operation but it uses a minimal amount of memory.
```

deployment.apps/python-cpu-load scaled

$ kubectl get pods -n cluster-autoscaler
NAME                               READY   STATUS    RESTARTS   AGE
python-cpu-load-59bfff47f4-9lgnq   1/1     Running   0          3m54s
python-cpu-load-59bfff47f4-c9ttt   1/1     Running   0          15s
python-cpu-load-59bfff47f4-d6trc   1/1     Running   0          3m52s
python-cpu-load-59bfff47f4-m4vcw   0/1     Pending   0          15s
python-cpu-load-59bfff47f4-md69d   0/1     Pending   0          15s
python-cpu-load-59bfff47f4-n8fgl   1/1     Running   0          15s
python-cpu-load-59bfff47f4-nbwqd   1/1     Running   0          15s
python-cpu-load-59bfff47f4-nbxcj   0/1     Pending   0          15s
python-cpu-load-59bfff47f4-rbstz   0/1     Pending   0          15s
python-cpu-load-59bfff47f4-tczf2   1/1     Running   0          15s
python-cpu-load-59bfff47f4-vzq2g   1/1     Running   0          3m50s
python-cpu-load-59bfff47f4-xqd72   1/1     Running   0          15s
```

As you can see then with `kubectl get nodepools`, the autoscaler detects capacity has been reached and asks for a new node:

```console
$ kubectl get nodepools

> kubectl get nodepools

Reviewer 2: the command is displayed two times? :)

Author: fix https://github.com/ovh/docs/pull/5940/commits/bcc1b4a09339b41ea5fb0aadddc70dc9d6d9c97d
